plexpy/webserve.py

```diff
@@ -251,6 +255,20 @@ def get_library_list(self, **kwargs):
         return json.dumps(library_list)
 
     @cherrypy.expose
+    @addtoapi()
+    def get_library_sections(self, **kwargs):
+
+        library_data = libraries.Libraries()
+        result = library_data.get_sections()
+
+        if result:
+            cherrypy.response.headers['Content-type'] = 'application/json'
+            return json.dumps(result)
+        else:
+            logger.warn(u"Unable to retrieve data for get_library_sections.")
+
+    @cherrypy.expose
+    @addtoapi()
     def refresh_libraries_list(self, **kwargs):
```

Comment:
Commands which don't return anything (such as `refresh_libraries_list`) return error in the API.

```
{"response": {"message": null, "data": {}, "result": "error"}}
```